assets/css/main.css

```diff
@@ -366,4 +366,15 @@ div.dataTables_scrollHead th:first-child {
   padding: 14px;
   text-align: center;
   color: #ffffff;
+}
+
+/* Modify the colvis column-visibility and -grouping buttons */
```

Comment:
Could you move these styles to be under https://github.com/nexB/aboutcode-manager/blob/9b9f3d5fd7dbea587b49cbbf8ce633f6354d8560/assets/css/main.css#L236-L243